test/unit/files/test_ftp.py

```diff
@@ -26,7 +26,7 @@ def test_file_source_ftp_specific():
         assert_realizes_contains(
             file_sources,
             test_url,
-            "This is ftp.gnu.org, the FTP server of the the GNU project.",
+            "This is ftp.gnu.org, the FTP server of the GNU project.",
```

Comment:
This string comes from the FTP server, please revert:

```suggestion
            "This is ftp.gnu.org, the FTP server of the the GNU project.",
```

Reply:
Thanks! Reverted